ipywidgets/__init__.py

```diff
@@ -20,33 +20,20 @@
 
 import os
 
-from IPython import get_ipython
 from ._version import version_info, __version__, __protocol_version__, __jupyter_widgets_controls_version__, __jupyter_widgets_base_version__
 from .widgets import *
 from traitlets import link, dlink
 
 
-def load_ipython_extension(ip):
```

Comment:
I'm not sure about the changes on the `__init__.py`, I didn't know what to keep, what to change, and what to remove.

Reply:
After in-person discussion with Sylvain, this doesn't seem to be useful. Maybe @jasongrout you have an idea? Do you think this can be removed? Or stay as an alias for `register_comm_target` for backward compatibility?